plugin.py

```diff
@@ -2,15 +2,17 @@
 import sublime
 import sys
 
-from LSP.plugin.core.typing import Any, Dict, List, Tuple
+from LSP.plugin.core.typing import Any, Dict, List, Optional, Tuple
 from lsp_utils import NpmClientHandler
+from sublime_lib import ActivityIndicator
 
 
 def plugin_loaded() -> None:
     LspPyrightPlugin.setup()
 
 
 def plugin_unloaded() -> None:
+    return
```

Comment:
Test code? :)

Reply:
Ah yes, also I wonder how you deal with this since saving `plugin.py` will trigger `plugin_unloaded()` and the server binary will be deleted and reinstall.

Reply:
I would deal with it the same way or just let it reinstall itself every time :).